conf/systemd.service

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment="JELLYFIN_PublishedServerUrl=https://__DOMAIN____PATH__"
```

Comment:
Same here, what's the doc for that ? It looks like it's related to docker based installs.

Reply:
It's the config to fix #141 

I wanted to add it to network.xml but upstream only wants ENV/CLI setting for this, no configuration file option, see https://github.com/jellyfin/jellyfin/issues/11564

Reply:
Alright. I finally found the doc : https://jellyfin.org/docs/general/administration/configuration#main-configuration